README.md

```diff
@@ -77,6 +78,19 @@ A curated list of awesome Java frameworks, libraries and software.
 
 - - -
 
+## ASCII/Console
+
+*Tools that ease console IO operations*
+
+* [ASCII Art](https://github.com/alcibiade/asciiart) - The tool for creating ASCII reports, multi-column tables, graphs. Supports for rendering images as strings.
```

Comment:
Don't use empty phrases like "The tool", "A simple tool" or "Java" since all this is implied. The CONTRIBUTION.md should list this I think.

Reply:
Doesn't exist anymore